### ISSUE TITLE
[IE11] Embedded Audio: Wave visuals not supported

### ISSUE THREAD

**Issue by gracecanfield** [vigetlabs/npm#273]
**Embedded audio:**
* [ ] The visual wave patterns aren't displaying in the audio embed. Examples:
  * Click into any of the audio spec examples on https://live-national-public-media-v2.pantheonsite.io/products/radio-credit-local/ ([screenshot](https://www.dropbox.com/s/cfos4vzxlonfu5t/Screenshot%202019-12-10%2013.27.01.png?dl=0))
  * View the audio spec samples on https://live-national-public-media-v2.pantheonsite.io/work/capitalone/ ([screenshot of current](https://www.dropbox.com/s/zemp0aqnzj97c47/Screenshot%202019-12-10%2013.26.23.png?dl=0)) 

**Note from Grace**
Chatted w/ @ten1seven - he investigated and this plugin just insn't supported in IE11. Functional for launch; the visuals just don't display while the audio is playing. If we wanted to revisit in the future, we could home-roll a progress bar with support from the audio script we're already using - this would take ~1 hr.

**Comment by gracecanfield** [comment]
@jeremyfrank Did some further investigation and it sounds like this actually just isn't possible without redesigning the page. @jeremyfrank Could you note the options here? For some reason I can't find them anywhere in slack.

**Comment by jeremyfrank** [comment]
I investigated an issue related to **overall page weight** of the podcast page being huge. It was due to the fact that each audio player needs to download its corresponding audio file in its entirety in order to generate the waveform visual. There weren't many options for addressing that:

1. Use the server to pre-process the audio files and generate a waveform file (text/xml?) and use that for the waveform visuals, and then lazyload the audio files when initiating playback. We quickly ruled this out due to hosting provider limitations (Pantheon).
2. Use a different audio player.
3. Leave it as is.
